autosklearn/util/backend.py

```diff
@@ -331,6 +331,16 @@ def load_datamanager(self) -> AbstractDataManager:
             with open(filepath, 'rb') as fh:
                 return pickle.load(fh)
 
+    def get_done_directory(self):
+        return os.path.join(self.internals_directory, 'done')
+
+    def note_numrun_as_done(self, seed, num_run):
+        done_directory = self.get_done_directory()
+        os.makedirs(done_directory, exist_ok=True)
+        done_path = os.path.join(done_directory, '%d_%d' % (seed, num_run))
```

Comment:
Hey @mfeurer, if I understand correctly, the file is not written to disk (yet probably). Do you think we want to be cautious and only create the done file if the path exists? Like doing an if os.path.exist(..): open(done_path, 'a').close()

Reply:
I'm not sure I can follow you here, so I try to answer all questions I can identify here:
1. the file is written to disk by `open('name', 'w')`. I just added a unit test to ensure this.
2. I think the directory is created in the 2nd call inside the function. Therefore, I'm not worried (yet) about it not being there